### ISSUE TITLE
Captcha still doesn't work.

### ISSUE THREAD

**Issue by lumitry** [adriankumpf/teslamate#1836]
**Describe the bug**

Captcha still doesn't work.  I can't log in.

**Expected behavior**

Log in successful.

**How to reproduce it (as minimally and precisely as possible)**:

1. Install (I did via Docker)
2. Go to localhost:4000
3. Attempt at logging in.

**Relevant entries from the logs**

```
2021-07-15 22:10:18.987 [info] GET https://auth.tesla.com/oauth2/v3/authorize -> 200 (356.739 ms)

2021-07-15 22:10:19.307 [info] GET https://auth.tesla.com/captcha -> 200 (317.283 ms)

2021-07-15 22:10:33.679 [info] GET https://auth.tesla.com/oauth2/v3/authorize -> 303 (734.742 ms)

2021-07-15 22:10:36.805 [info] GET https://auth.tesla.cn/oauth2/v3/authorize?login_hint=[EMAILWASHERE]&client_id=ownerapi&redirect_uri=https%3A%2F%2Fauth.tesla.com%2Fvoid%2Fcallback&response_type=code&scope=openid+email+offline_access&code_challenge=d9uTHDNpA38Mg1k7nTu_0HrGi8iA_jMRz70N7CxDrQw&code_challenge_method=S256&state=Ccz7TZMW0UzcZGy -> 200 (3125.805 ms)

2021-07-15 22:10:37.054 [info] GET https://auth.tesla.com/captcha -> 200 (246.062 ms)

2021-07-15 22:10:37.592 [info] POST https://auth.tesla.cn/oauth2/v3/authorize -> 200 (537.113 ms)

2021-07-15 22:10:37.772 [info] GET https://auth.tesla.com/oauth2/v3/authorize -> 200 (179.214 ms)

2021-07-15 22:10:37.929 [info] GET https://auth.tesla.com/captcha -> 200 (154.382 ms)
```

**Screenshots**

![screenshot](https://user-images.githubusercontent.com/53492381/125864668-f6199728-ccac-416f-ba0e-bad92a55ab52.png)

**Environment**

- TeslaMate version: 1.23.6
- Type of installation: **Docker**
- OS TeslaMate is installed on: Windows
- User OS & Browser: Windows (Same machine). Vivaldi
- Others: No captcha abuse from my IP. In screenshot/logs I used a fake password and email.

Sidenote:
All of the issues involving this are closed and the changelogs say it's fixed, but it's still not working for me. I have no clue why I would be any different. Any help here is appreciated, but I can't find any other reason why this would still be broken.

**Comment by arthur-rtzr** [comment]
+1

**Comment by danijelgr** [comment]
+1

**Comment by nanara1119** [comment]
+1

**Comment by fjongfasong2** [comment]
+1

**Comment by nwadams** [comment]
@adriankumpf I was able to pull `edge` a few minutes ago and was able to log in successfully. Thank you for the fix.

**Comment by adriankumpf** [comment]
Fixed in v1.23.7
